.github/workflows/workflow.yaml

```diff
@@ -53,3 +53,18 @@ jobs:
           name: logs
           path: |
             logs-*.json
+  require_suites:
```

Comment:
Any reason to have these as separate actions instead of one?

Reply:
This way I can know what the problem is without having to look at the log output directly.  I'd just see the failed job and know what to go debug.

They also run in parallel, though the speedup likely isn't meaningful.

Reply:
Sounds good to me!